Simulator/Core/Simulator.cpp

```diff
@@ -83,7 +84,16 @@ void Simulator::loadParameters() {
    ParameterManager::getInstance().getIntByXpath("//SimParams/numEpochs/text()", numEpochs_);
    ParameterManager::getInstance().getIntByXpath("//SimConfig/maxFiringRate/text()", maxFiringRate_);
    ParameterManager::getInstance().getIntByXpath("//SimConfig/maxEdgesPerVertex/text()", maxEdgesPerVertex_);
-   ParameterManager::getInstance().getLongByXpath("//Seed/value/text()", seed_);
+
+   // Instantiate rng object 
+   string type;
+   ParameterManager::getInstance().getStringByXpath("//RNGConfig/NoiseRNGParams/@class", type);
+   rgNormrnd = RNGFactory::getInstance()->createRNG(type);
+
+   ParameterManager::getInstance().getLongByXpath("//RNGConfig/InitRNGParams/Seed/text()", rngSeed_);
+   ParameterManager::getInstance().getLongByXpath("//RNGConfig/NoiseRNGParams/Seed/text()", seed_);
+   rgNormrnd->seed(seed_);
+   rng.seed(rngSeed_);
```

Comment:
Let's also give `rng` a clearer/less generic name, consistent with naming conventions.

Reply:
Renamed `rng` to `initRNG`